tests/postgres_tests/test_hstore.py

```diff
@@ -37,6 +39,17 @@ def test_value_null(self):
         reloaded = HStoreModel.objects.get()
         self.assertEqual(reloaded.field, value)
 
+    def test_key_val_cast_to_string(self):
+        value = {'a': 1, 'b': 'B', 2: 'c', 'd': 'ê'}
+        expected_value = {'a': '1', 'b': 'B', '2': 'c', 'd': force_text('ê')}
```

Comment:
I think `force_text()` isn't needed but rather adding `from __future__ import unicode_literals` to this file.